src/parser/jessiecode.js

```diff
@@ -678,16 +678,19 @@ JXG.extend(
                 this.replaceNames(node.children[1]);
 
                 /** @ignore */
-                fun = (function ($jc$) {
+                fun = (function (jc) {
```

Comment:
I renamed the variable too just to be sure in tests that any `$jc$` in the `eval`ed string is coming from the surrounding scope. If we accidentally back out the rest of the change later the tests will fail and catch it.